docs/minio-admin-complete-guide.md

```diff
@@ -454,3 +456,25 @@ mc admin heal -r myminio/mybucket
 ```sh
 mc admin heal -r myminio/mybucket/myobjectprefix
 ```
+
+<a name="top"></a>
+### Command `top` - provide top like statistics for Minio
+NOTE: This command is only applicable for distributed Minio setup.
+
+```
+NAME:
+  mc admin top - provide top like statistics for Minio
+
+FLAGS:
+  --help, -h                    show help
+
+COMMANDS:
+  locks  Get list of oldest locks on a Minio server/cluster.
```

Comment:
"Get list of the 10 oldest locks".  Might also want to remove the 'server' part and just say cluster, otherwise seems to contradict the statement above that it only works on dist. setups.

Reply:
done